docker/ray-deps/Dockerfile

```diff
@@ -1,9 +1,18 @@
 ARG GPU=""
+ARG AUTOSCALER="autoscaler"
```

Comment:
Let's add a comment saying if this is removed, then autoscaler dependencies won't be installed

Reply:
Okay!